src/config/graphical-session/fonts.md

```diff
@@ -1,5 +1,8 @@
 # Fonts
 
+[xbps-query(1)](../../xbps/index.md#finding-files-and-packages) can be used to
+list font-related packages. Many fonts are packaged with the `font-` prefix.
```

Comment:
```suggestion
A number of fonts and font collections are [available from
XBPS](../../xbps/index.md#finding-files-and-packages). If you are unsure which
fonts to install, `dejavu-fonts-ttf` and `xorg-fonts` are good basic options.
`noto-fonts-ttf` provides fonts that contain characters for many languages and
scripts. `noto-fonts-cjk` provides fonts for Chinese, Japanese, and Korean
characters. `noto-fonts-emoji` provides emojis. `nerd-fonts` provides a number
of fonts with special characters added.

Fonts not available from XBPS can be installed manually to `/usr/share/fonts`
(system-wide) or `~/.local/share/fonts` (per-user).
```

Reply:
Is (from line 4 to 9)

```diff
+ XBPS](../../xbps/index.md#finding-files-and-packages). `dejavu-fonts-ttf` or `xorg-fonts`
+ are a good baseline if you're unsure of what to pick.
+ `noto-fonts-ttf` contains fonts for many languages and scripts.
+ `noto-fonts-cjk` extends this with fonts for Chinese, Japanese, and Korean, and
+ `noto-fonts-emoji` provides emojis.
+ `nerd-fonts` provides a number of fonts with special characters like custom icons included.
+ Many other font packages begin with `font-`.
```

(I'm fine dropping the last line if it's unnecessary)

adequate or would you prefer that suggestion verbatim? It's pedantic but I find this wording more varied and a bit easier to understand. (Written without ensuring correct column wrapping.)

Reply:
my thinking is it's reasonable for the user to understand that `xbps-query - Rs font` would be how you find fonts, so we don't have to write it out.